### PR TITLE
chore: remove client columns

### PR DESCRIPTION
**Pull request by Bekkalizer**

### CODE REVIEW

File: i18n/i18n_app.properties [hunk 1]
```diff
@@ -682,6 +682,4 @@ attribute_to_search=attribute to search
 attributes_to_search=attributes to search
 Yes=Yes
 No=No
-save failed=Save failed
-last_date_in_isolation=Last date in isolation
-last_date_in_quarantine=Last date in quarantine
\ No newline at end of file
+save failed=Save failed
```
Comment (helgeskjellevik) [inline i18n/i18n_app.properties:685]: mangler det underscore her?

File: i18n/i18n_app_nb.properties [hunk 2]
```diff
@@ -682,6 +682,4 @@ attribute_to_search=felt for å søke
 attributes_to_search=felter for å søke
 Yes=Ja
 No=Nei
-save failed=Kunne ikke lagre
-last_date_in_isolation=Siste isolasjonsdag
-last_date_in_quarantine=Siste karantenedag
\ No newline at end of file
+save failed=Kunne ikke lagre
```
Comment (helgeskjellevik) [inline i18n/i18n_app_nb.properties:685]: og her

File: scripts/services.js [hunk 3]
```diff
@@ -2206,17 +2206,15 @@ var trackerCaptureServices = angular.module('trackerCaptureServices', ['ngResour
...
-
                     if(grid.headers[grid.headers.length-1].column == 'TransferStatus'){
                         //entity.followUp =  entity.followUp || row[row.length-1] == "ACTIVE";
                     }
                     
                     for (var i = 7; i < row.length; i++) {
                         if (row[i] && row[i] !== '') {
                             var val = row[i];
+                            if(!grid.headers[i]) 
+                            {debugger};
```
Comment (helgeskjellevik) [inline scripts/services.js:2217]: debugger?

Comment (Bekkalizer) [reply scripts/services.js:2217]: Nice catch. This was a bit of debug code.

**Review by helgeskjellevik — APPROVED** [review]
lgtm

**Review by helgeskjellevik — APPROVED** [review]
lgtm

File: i18n/i18n_app.properties (hunk 1 above)
Comment (Bekkalizer) [reply i18n/i18n_app.properties]: Fixed - this was unrelated to the current code change.

**Pull request merged**